src/ray/core_worker/core_worker.cc

```diff
@@ -2867,6 +2874,7 @@ void CoreWorker::HandleReportGeneratorItemReturns(
     rpc::ReportGeneratorItemReturnsRequest request,
     rpc::ReportGeneratorItemReturnsReply *reply,
     rpc::SendReplyCallback send_reply_callback) {
+  WaitUntilInitialized();
```

Comment:
Is there any way to modify a macro instead here? core_worker_server.h. Seems to be prone to error...